sanic/response.py

```diff
@@ -149,6 +159,8 @@ async def send(
             if hasattr(data, "encode")
             else data or b""
         )
+        if self.auto_content_length and "content-length" not in self.headers:
```

Comment:
Why are we adding this here on the base? This is something that will already be added when needed: https://github.com/sanic-org/sanic/blob/86ae5f981cbe64fd85bee34fee3989b009e90dd2/sanic/http.py#L300

Reply:
Oh, that's true. Removed auto content length in response.py and let Http handle it. Thanks!